uawidgets/new_node_dialogs.py

```diff
@@ -54,6 +61,8 @@ def __init__(self, parent, title, server):
 
     def _store_state(self):
         self.settings.setValue("last_namespace", self.nsComboBox.currentIndex())
+        self.settings.setValue("last_auto_nodeid_state", self.nodeidCheckBox.isChecked())
```

Comment:
the name of settings will get is into trouble. What does False eller True mean. rename it to something like. 'nodeid_widget_visible'
Then the default should probably be unvisible. It requires to know quite a lot about nodeid